bitnami/mariadb/Chart.yaml

```diff
@@ -26,4 +26,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-mariadb
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 11.1.1
+version: 11.1.2
```

Comment:
Version 11.1.2 has already been released, could you please update this to version `11.1.3`?